src/pydagmc/dagnav.py

```diff
@@ -228,7 +228,11 @@ def _check_category_and_dimension(self):
             raise ValueError(f"{identifier} has no category or geom_dimension tags assigned.")
 
     def __eq__(self, other):
-        return self.model == other.model and self.handle == other.handle
+        if type(other) != type(self):
+            return False
+        return type(other) == type(self) and \
```

Comment:
Isn't this first condition redundant with the previous one?

Reply:
Whoops! Thought I'd removed that. Thanks!